usecases/objects/auto_schema.go

```diff
@@ -83,11 +83,16 @@ func (m *autoSchemaManager) autoSchema(ctx context.Context, principal *models.Pr
 			return err
 		}
 		if schemaClass == nil {
-			return m.createClass(ctx, principal, object.Class, properties)
+			if err := m.createClass(ctx, principal, object.Class, properties); err != nil {
+				return err
+			}
+			continue
 		}
```

Comment:
**praise:** nice fix 👍🏼 , i had it on my todo because atm `autoSchema` is called with one object in the places and wanted to change once refactoring batchAdd